index.html

```diff
@@ -2891,6 +2891,40 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
+        </h3>
+        <p>
+          The <dfn id="member-iarc_rating_id"><code>iarc_rating_id</code> member</dfn>
+          is a <a>string</a> that represents an ID value of the IARC rating of the 
```

Comment:
It is not like it is hidden, but their sites it just not very precise at explaining things :-) and you will have to go through a ratings authority anyway, so they will probably be the ones ending up giving you the string - but I cannot find docs explaining how that string looks

https://www.globalratings.com/for-developers.aspx